robottelo/hosts.py

```diff
@@ -2545,3 +2549,42 @@ def remove_user_from_usergroup(self, member_username, member_group):
         )
         if result.status != 0:
             raise IPAHostError('Failed to remove the user from user group')
+
+
+class ProxyHost(Host):
+    """Class representing HTTP Proxy host"""
+
+    def __init__(self, url, **kwargs):
+        self._conf_dir = '/etc/squid/'
```

Comment:
```suggestion
        self._conf_dir = '/etc/squid'
```
Strings below would look better if you did this and then called `f'{self._conf_dir}/passwd'`